docs/sequence_diagrams/notification-adjustmentRuleValidation.plantuml

```diff
@@ -0,0 +1,94 @@
+@startuml
```

Comment:
Please, change what is applicable from above comments.

Reply:
Same as previous.